src/docker/registry.nim

```diff
@@ -269,20 +448,28 @@ proc manifestHead*(image: DockerImage): DockerDigestedJson =
   let kind = CONTENT_TYPE_MAPPING[contentType]
   return newDockerDigestedJson("{}", digest, contentType, kind)
 
-proc manifestGet*(image: DockerImage, accept: string): DockerDigestedJson =
+proc manifestGet*(image:  DockerImage,
+                  accept: string,
+                  use =   RegistryUse.ReadOnly,
+                  ): DockerDigestedJson =
   let
     kind          = CONTENT_TYPE_MAPPING[accept]
     (_, response) = image.request(
+      use        = use,
       httpMethod = HttpGet,
       path       = "/manifests/" & image.imageRef,
       accept     = accept,
     )
   return newDockerDigestedJson(response.body(), image.imageRef, accept, kind)
 
-proc layerGet*(image: DockerImage, accept: string): DockerDigestedJson =
+proc layerGet*(image:  DockerImage,
+               accept: string,
+               use =   RegistryUse.ReadOnly,
+               ): DockerDigestedJson =
```

Comment:
nit: closing paren and return type should be raised to next line to be consistent with the rest of the PR.

Reply:
the pattern we seem to use is:

sing-line proc signature:

```nim
proc foo(a: string): int=
```

when there are a lot of params we break it up into multiple lines:

```nim
proc foo(a: string,
         b: string,
         ): int =
```

eventually will be nice to get proper autoformatter involved too. at some nim switched recommeded format from hanging to:

```nim
proc foo(
  a: string,
  b: string,
): int =
```

I actually prefer that as it matches `black` more

Reply:
i like the last option best as well. keeps the interesting bits away from too much indentation.